### DISCUSSION TITLE
active pseudo-classes reset styles, which will conflict with the original set styles.

### DISCUSSION THREAD

**Discussion by ryanli7426** [tailwindlabs/tailwindcss#12233] [Help] [answered]
Set the border color for "span" by default, then use the "active" pseudo-class to change the border color. After the setting is done without any operation, the border color is automatically set to the border color changed by the "active" pseudo-class.

Code:
     <span class="inline-block w-38 h-28 border border-solid rounded-s-lg border-r-0 border-DBDDE2 text-center leading-28
			hover: cursor-pointer active: border-violet-700">
		<UnorderedListOutlined class="w-15 h-15"/>
     </span>

**Comment by alex-krasikau** [comment]
Hello, @youvergood7426!

Can you please provide a reproduction example? I tried it myself, but I don't experience the same issue - https://play.tailwindcss.com/5orwj5t51S.

**Comment by ryanli7426** [comment]
@alex-krasikau  https://play.tailwindcss.com/hnalRKxTBz
thank you so muchv！

**Comment by alex-krasikau** [comment]
> @alex-krasikau https://play.tailwindcss.com/hnalRKxTBz thank you so muchv！

Don't put a space between pseudo-class modifier and the class utility. That's the reason.

**Comment by ryanli7426** [comment]
> > @alex-krasikau https://play.tailwindcss.com/hnalRKxTBz thank you so muchv！
> 
> Don't put a space between pseudo-class modifier and the class utility. That's the reason.
This is not good enough, and if you subsequently add other attributes, you will have the same problem.
https://play.tailwindcss.com/YbWYUmz2Cx

**Reply by alex-krasikau** [reply, marked as answer]
This is the way CSS works, not Tailwind, you should avoid using conflicting utils on the same element. The order in which they get applied is defined by the order of definition in CSS file, not by the order of definition in class name.
